<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FSExplorer</class>
 <widget class="QMainWindow" name="FSExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>461</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>461</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>461</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FS - Explorer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/Resources/logo.png</normaloff>:/Resources/logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>335</y>
      <width>791</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.0, cy:0.0, radius:0.0, fx:0.0, fy:0.0, stop:0 rgba(0, 0, 0, 0), stop:0.35 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:0.425 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));


</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Disco Duro:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdTamDisco">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="numDigits">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
     <property name="intValue" stdset="0">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Inicio de bloque usuarios:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdUsuarios">
     <property name="geometry">
      <rect>
       <x>378</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qradialgradient(spread:pad, cx:0.0, cy:0.0, radius:0.0, fx:0.0, fy:0.0, stop:0 rgba(0, 0, 0, 0), stop:0.35 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:0.425 rgba(0, 0, 0, 0), stop:0.0 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0));</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="numDigits">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Inicio de bloque  mapa bits:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdMapabits">
     <property name="geometry">
      <rect>
       <x>378</x>
       <y>60</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="numDigits">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>No. Cluster:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNoCluster">
     <property name="geometry">
      <rect>
       <x>378</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="numDigits">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Inicio de bloque directorios:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="LcdDirectorios">
     <property name="geometry">
      <rect>
       <x>635</x>
       <y>20</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="numDigits">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Inicio de bloque de datos:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdBloqueDatos">
     <property name="geometry">
      <rect>
       <x>635</x>
       <y>60</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="numDigits">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Espacio Disponible:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdEspacioDisponible">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="numDigits">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>53</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Espacio Usado:</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdEspacioUsado">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>57</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="numDigits">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QCommandLinkButton" name="btnUsuario">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>134</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuarios</string>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/Resources/config-users.png</normaloff>:/Resources/config-users.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+U</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="btnFormato">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Formato</string>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/Resources/harddrive2.png</normaloff>:/Resources/harddrive2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+F</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="btnFilesSystem">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>269</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Archivos</string>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/Resources/system-file-manager.png</normaloff>:/Resources/system-file-manager.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+A</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="blogo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>91</width>
      <height>131</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="recursos.qrc">
      <normaloff>:/Resources/logo.png</normaloff>
      <disabledoff>:/Resources/logo.png</disabledoff>:/Resources/logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>94</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>800</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Resources/barra.png);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
